src/component/ListContents.js

```diff
@@ -62,7 +62,6 @@ const ListContents = ({ listData, firestore, token }) => {
             <div
               className={calculateIfPurchased(item)}
               onClick={handlePurchase}
-              id={item.id}
```

Comment:
I was getting a few failing tests and I think it's because of this line. I think you should be able to take out the `id` in the `li` without issues if we're trying to reduce on `id`s?

Reply:
Oh just kidding - taking out either one causes Cypress errors 😆

Reply:
Doing this: `<li id={item.id + '-li'}...` on line 61 and changing the test to look for `#banana-li` worked for me!

Reply:
I am finding that I get an error when I try to purchase an item that is added to the list: 
```
FirebaseError: Function CollectionReference.doc() requires its first argument to be of type non-empty string, but it was: ""
handlePurchase
c:/Coding/Projects/Collab-Lab/tcl-2-smart-shopping-list/src/component/ListContents.js:19
  16 | const handlePurchase = event => {
  17 |   let itemId = event.target.id;
  18 |   event.preventDefault();
> 19 |   itemsDocRef
     | ^  20 |     .doc(itemId)
  21 |     .get()
  22 |     .then(doc => {
```

Reply:
Anyone else having this problem?

Reply:
Yah, and that was when the test was failing for me too - when it went to purchase an item. Doing the fix above should fix that error though!

Reply:
hey @gunderodd , sorry a little confused. You got that error in the test? Or the application?

Reply:
ahh now i follow, sorry @gunderodd 

@mindyzwan question, if i put that id back into the div all my tests pass and the application does not bug out, but is that not the correct way to make it work? i think im lost.

Reply:
Mm - I think the issue is that id values should be unique on a page. If you run a page through Axe with two elements having the same id, it pops up with an issue. So doing the `item.id + 'li'` makes the `li` element have a different id value from the `div`

Reply:
Hey guys sorryyyyy for the confusion! I removed the div id for accessibility since we had duplicate id's. I did some checks and it seemed to be working fine, but my checks were not thorough enough. I used your suggestions above @mindyzwan and everything seems to be a-okay now. Let me know if any of you notice any other issues.

Reply:
ahhh ha, gotcha @mindyzwan thanks for clearing it up

Reply:
Great discussions team 👏